CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [0.13.2] - 2020-12-27
+### Changed
+- Added libmdns::Service property called `dns_service` to WebThingServer
+- WebThingServer::new() now takes an additional property (`dns_service`) which can be `None`
```

Comment:
This is no longer true, so it can be removed.

Reply:
Fixed: https://github.com/WebThingsIO/webthing-rust/pull/54/commits/233fab92055d26ee577354682873b3f6beb0ff93